### ISSUE TITLE
Turn off "store activities on my watch"

### ISSUE THREAD

**Issue by Inks27** [scubajorgen/TomTomWatch#50]
Is it posible to turn off "store activities on my watch" with your program ?

**Comment by scubajorgen** [comment]
What exactly do you mean?  
What is the feature 'store activities on my watch'? Storing GPS tracks on the watch? What is the use of the watch when you switch this off?

**Comment by Inks27** [comment]
My old friend use only distance and time from years :)

**Comment by Inks27** [comment]
for years :)

**Comment by scubajorgen** [comment]
As far as I know this is not possible. What you can do is periodically erase tracks and reports from the watch...

**Comment by Inks27** [comment]
I have been using the program recently and everything works fine.
However, my friend would be free from the need to reset this function on his Windows XP computer used only for this purpose :)
I will install your program for him and he will be happy. Thank you for your great work.

**Comment by Inks27** [comment]
However, this feature could be set in the app itself, which TomTom disabled.
TomTom advice:

What will happen?
You can store activities on your watch, however we suggest you switch off this functionality as you will not be able to delete this after the app has been discontinued. To avoid your storage filling up we suggest that you do not store activities on the product itself.

https://help.tomtom.com/hc/en-hu/articles/11748276052370-TomTom-Sports-discontinuation#h_01H90NECE7A5YFWBK70B6DKMJK

**Comment by scubajorgen** [comment]
Yup, I can imagine that if you keep using the watch without connecting it will fill up the storage. It will take some time though. However my experience is that it starts to respond slower when starting a new activity when the memory gets fuller.

I don't find the setting in the **preference file**. So it must be in the **settings file** I guess. Both can be requested when you enable the debug settings of the watch. Some settings have been identified like age and weight. However, most of the ~800 settings are still unknown.
